CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## vNext
+- Fixed preflight check misaligned issue
```

Comment:
Follow the single format for "Fix" (check you previous message).

Reply:
The previous message has the wrong format as we used earlier `-ed` ending. Do you want to switch to a format without `-ed` ending?